src/lightning-address.ts

```diff
@@ -85,8 +87,9 @@ export default class LightningAddress {
     try {
       const nostrResult = await fetch(this.nostrUrl());
       const data = await nostrResult.json();
-      this.nostrData = data.names;
-      this.nostrPubkey = this.nostrData?.[this.username];
+      this.nostrData = data;
+      this.nostrPubkey = this.nostrData.names?.[this.username];
+      this.nostrRelays = this.nostrPubkey ? this.nostrData.relays?.[this.nostrPubkey] : [];
```

Comment:
I think this means if we have a `nostrPubkey` but the `nostrData` does not return anything there (relays is optional) then `nostrRelays` becomes null/undefined 
but if `nostrPubkey` is not there then we return `[]`

wondering what's the best default. [] or undefined? if we don't have anything there. 

@rolznz do you have thoughts on this?

Reply:
I think it should be consistent for both cases (no relays for a pubkey or no pubkey set).

In my opinion undefined is better but an empty array is also fine.

Reply:
changed it to be undefined